untrusted/quote_verify/src/lib.rs

```diff
@@ -16,8 +16,8 @@ const SIGNATURE_COMPONENT_SIZE: usize = 32;
 // The starting byte of the quote report.  The *QE Report* member of the Quote
 // Signature Data Structure.  Table 4 of
 // https://download.01.org/intel-sgx/latest/dcap-latest/linux/docs/Intel_SGX_ECDSA_QuoteLibReference_DCAP_API.pdf
-// TODO as more of the quote structure is brought in, derive the start by summation
-//  of previous type/member sizes.
+// TODO as more of the quote structure is brought in, derive the start by
+// summation  of previous type/member sizes.
 const QUOTING_ENCLAVE_REPORT_START: usize = 0x234;
```

Comment:
no, 0x234(hex) == 564(decimal).  I'm fine converting to dec if we want, but I'm hoping to clean this up with
>  Extract the quote bytes to a Rust type, need to validate the bytes as they are unpacked.

from #17.

Speaking of do we want to convert task lists to issues for easier linking or take it on a case by case basis?

Reply:
OK, so where I'm going with this is: we need to actually expose all the types that are contained in a quote as their own types, because we're going to be examining all of them, and we need to be able to use them independently.

Reply:
Moved the 
> extract quote bytes

to a  specific issue #25 to ensure it doesn't get glossed over and so it's easier to further break down